<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbFunction">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>-1</y>
      <width>1261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { 
color: #7b5701;
 } </string>
    </property>
    <property name="text">
     <string>Vietnamese Sign Language Recognition System</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbNotify">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Thông Báo</string>
    </property>
    <widget class="QLabel" name="lbNotify">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>521</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbCountDownTime">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>541</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Thời Gian</string>
    </property>
    <widget class="QLabel" name="lbCountDownTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>521</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbFrame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>611</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } 



</string>
    </property>
    <property name="title">
     <string>Hình ảnh bàn tay</string>
    </property>
    <widget class="QPushButton" name="btnLearning">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnLearning {
background-color: white;
}</string>
     </property>
     <property name="text">
      <string>Học</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRecognition">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnRecognition {
background-color: white;
}</string>
     </property>
     <property name="text">
      <string>Nhận Dạng</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbFrame">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnRecognition {
background-color: white;
}</string>
     </property>
     <property name="text">
      <string>Lên</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDown">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>130</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#btnLearning {
background-color: white;
}</string>
     </property>
     <property name="text">
      <string>Xuống</string>
     </property>
    </widget>
    <zorder>lbFrame</zorder>
    <zorder>btnLearning</zorder>
    <zorder>btnRecognition</zorder>
    <zorder>btnUp</zorder>
    <zorder>btnDown</zorder>
   </widget>
   <widget class="QGroupBox" name="gbRecognitionNotify">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Thông Báo</string>
    </property>
    <widget class="QLabel" name="lbRecognitionNotify">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Hệ thống sẽ lưu lại kết quả nhận dạng sau 3 giây!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbRecognitionTimer">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>541</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Thời Gian</string>
    </property>
    <widget class="QLabel" name="lbRecognitionTimer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>521</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbRecognitionResult">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>611</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Kết quả nhận dạng</string>
    </property>
    <widget class="QLabel" name="lbRecognitionResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>591</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbRecognitionContent">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>541</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Nội dung toàn bộ</string>
    </property>
    <widget class="QLabel" name="lbRecognitionContent">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>521</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbTestingResult">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>611</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Kết quả kiểm tra</string>
    </property>
    <widget class="QLabel" name="lbTestingResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>591</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#lbTestingResult {
    font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbLearningResult">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>611</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Kết quả nhận dạng</string>
    </property>
    <widget class="QLabel" name="lbLearningResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>591</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbLearningNotify">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Thông Báo</string>
    </property>
    <widget class="QLabel" name="lbLearningNotify">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>511</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Hãy đưa kí hiệu trong hướng dẫn vào vùng mũi tên lên xuống để thay đổi từ được chọn</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbListWord">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>541</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Bảng ngôn ngữ câm</string>
    </property>
    <widget class="QListWidget" name="lwWord">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>201</width>
       <height>341</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbLearningImage">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>281</width>
       <height>281</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbTutorial">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>541</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
    font-size: 18px;
    font-weight: bold;
color: #7b5701;
 } </string>
    </property>
    <property name="title">
     <string>Hình ảnh hướng dẫn</string>
    </property>
    <widget class="QLabel" name="lbTutorialImage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>501</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
